### ISSUE TITLE
WSDL + SOAP support status

### ISSUE THREAD

**Issue by SamuelYvon** [tefra/xsdata#716]
We (Garda World) are looking at using xs-data [pydantic] for WSDL / SOAP interactions. Specifically,
 
> The implementation is experimental and currently only WSDL 1.1 and SOAP 1.1 bindings have been implemented.

What level of experimental is experimental? Are there open issues / tasks that can be helped with? Is the API stable?

**Comment by tefra** [comment]
When I first introduced it I had access to a couple wsdl apis and it was working just fine.

Here is one of the samples
https://github.com/tefra/xsdata-samples/tree/master/travelport

I am quite confident about the models and the generated service ports, this year a few issues with rpc operations were also fixed. The client was something that I put together very quickly, the whole transport layer is based on requests and it's quite easy for someone to extend or replace it completely.

The reason it has stayed experimental is that not many users are using it and in the past a few years only a handful of issues were submitted. Which could be a good thing but most likely it's because not many people are using it.

I will be glad to help/fix any issues that may come up.
